### PR TITLE
MUMUP-2886 adding name transitions and descriptions

### PR DESCRIPTION
**Pull request by davidmsibley**
Proposing project name changes for incubation

----

Contributor License Agreement adherence:

- [x] This Contribution is under the terms of Individual [Contributor License Agreements][] (and also Corporate Contributor License Agreements to the extent applicable) appearing in the [Apereo CLA roster][].

[Apereo CLA roster]: http://licensing.apereo.org/completed-clas
[Contributor License Agreements]: https://www.apereo.org/licensing/agreements

### CODE REVIEW

File: docs/apereo-incubation.md [hunk 1]
```diff
@@ -48,7 +48,21 @@ Next actions:
 
 Next actions:
 
-+ New software product names that better reflect nature and do not infringe on others' trademarks. Probably a functional name describing what the product is, with "uPortal" brand attached. As in "uPortal Foo Sprocket".
++ New software product names that better reflect nature and do not infringe on others' trademarks.
++ Names should reflect uPortal commitment and represent how the product should
+be used.
++ Proposed names:
+  + `uw-frame` → `uPortal-app-framework`
```
Comment (ChristianMurphy) [inline docs/apereo-incubation.md]: +1

File: docs/apereo-incubation.md [hunk 2]
```diff
@@ -48,7 +48,21 @@ Next actions:
...
+    + The uPortal App Framework will house common functionality to be leveraged
+      by uPortal Apps.
+    + Institutions can provide a custom `uPortal-app-framework-config`
+    + Developers can create their own uPortal Apps drawing from common
+      configuration, style and functionality.
+  + `angularjs-portal` → `uPortal-home`
```
Comment (ChristianMurphy) [inline docs/apereo-incubation.md]: +0

**Comment by davidmsibley** [comment]
Good question in the uportal-dev thread: https://groups.google.com/a/apereo.org/forum/#!topic/uportal-dev/BXOjNeuTC7E

Why uPortal Home?
We were looking to describe the product we were delivering rather than the technology it uses.  This way we can be inclusive to new technologies and designs

**Pull request merged**